liquibase-integration-tests/src/test/groovy/liquibase/command/IncludeAllPreconditionsTest.groovy

```diff
@@ -0,0 +1,59 @@
+package liquibase.command
+
+import liquibase.Scope
+import liquibase.changelog.ChangeLogParameters
+import liquibase.command.core.UpdateCommandStep
+import liquibase.command.core.helpers.DatabaseChangelogCommandStep
+import liquibase.command.core.helpers.DbUrlConnectionArgumentsCommandStep
+import liquibase.command.util.CommandUtil
+import liquibase.extension.testing.testsystem.DatabaseTestSystem
+import liquibase.extension.testing.testsystem.TestSystemFactory
+import liquibase.extension.testing.testsystem.spock.LiquibaseIntegrationTest
+import liquibase.resource.SearchPathResourceAccessor
+import liquibase.ui.ConsoleUIService
+import spock.lang.Shared
+import spock.lang.Specification
+
+/**
+ * IncludeAll preconditions test
+ * @author Edoardo Patti
+ */
+@LiquibaseIntegrationTest
+class IncludeAllPreconditionsTest extends Specification {
+
+    @Shared
+    private DatabaseTestSystem h2 = Scope.currentScope.getSingleton(TestSystemFactory).getTestSystem("h2") as DatabaseTestSystem
+
+    def "run include with preconditions fail option WARN"() {
```

Comment:
I think we can also add a few more scenarios for the other options as well. If you agree I can add them if you wish.

Thanks,
Daniel.

Reply:
Yes, feel free to add them if you want, but they are already included [here](https://github.com/liquibase/liquibase/pull/6440/files/7dcca1d48479612786fbdd003657a9788a9cd7cc#diff-55f0516670210a2314bfe62e9f2479ffd2999fc1634894eab26e682f0de72eee). Preconditions flow executes the same code for both include and includeAll.

Reply:
You are right that we are already testing these cases for include. I just added two more for `includeAll` for testing `onFail` property with `CONTINUE` and `HALT` configurations.